app-text/calibre/calibre-6.29.0.ebuild

```diff
@@ -0,0 +1,255 @@
+	# wash anyway. The only real solution here is to package rapydscript-ng.
+	#
+	# We do not need it at build time, and *no one* needs it at install time.
+	# Delete the cruft.
+	rm -r resources/rapydscript/ || die
+}
+
+src_compile() {
+	# TODO: get qmake called by setup.py to respect CC and CXX too
+	tc-export CC CXX
+
+	# bug 821871
+	local MY_LIBDIR="${ESYSROOT}/usr/$(get_libdir)"
+	export FT_LIB_DIR="${MY_LIBDIR}" HUNSPELL_LIB_DIR="${MY_LIBDIR}" PODOFO_LIB_DIR="${MY_LIBDIR}"
+	export QMAKE="$(qt6_get_bindir)/qmake"
+
+	edo ${EPYTHON} setup.py build
+	edo ${EPYTHON} setup.py gui
+
+	# A few different resources are bundled in the distfile by default, because
+	# not all systems necessarily have them. We un-vendor them, using the
+	# upstream integrated approach if possible. See setup/revendor.py and
+	# consider migrating other resources to this if they do not use it, in
+	# *preference* over manual rm'ing.
+	edo ${EPYTHON} setup.py liberation_fonts \
+		--path-to-liberation_fonts "${EPREFIX}"/usr/share/fonts/liberation-fonts \
+		--system-liberation_fonts
+	if use system-mathjax; then
+		edo ${EPYTHON} setup.py mathjax --path-to-mathjax /usr/share/mathjax --system-mathjax
+		edo ${EPYTHON} setup.py rapydscript
+	fi
+}
+
+src_test() {
+	# Skipped tests:
+	local _test_excludes=(
+		# unpackaged Python dependency: py7zr
+		7z
+		# unpackaged Python dependency: pyzstd
+		test_zstd
+		# tests if a completely unused module is bundled
+		pycryptodome
+
+		$(usev !speech speech_dispatcher)
+		$(usev !unrar test_unrar)
+
+		# undocumented reasons
+		test_mem_leaks
+		test_searching
+	)
+
+	edo ${PYTHON} setup.py test "${_test_excludes[@]/#/--exclude-test-name=}"
+}
+
+src_install() {
+	# Bug #352625 - Some LANGUAGE values can trigger the following ValueError:
+	#   File "/usr/lib/python2.6/locale.py", line 486, in getdefaultlocale
+	#    return _parse_localename(localename)
+	#  File "/usr/lib/python2.6/locale.py", line 418, in _parse_localename
+	#    raise ValueError, 'unknown locale: %s' % localename
+	#ValueError: unknown locale: 46
+	export -n LANG LANGUAGE ${!LC_*}
+	export LC_ALL=C.utf8 #709682
+
+	# Bug #295672 - Avoid sandbox violation in ~/.config by forcing
+	# variables to point to our fake temporary $HOME.
+	export HOME="${T}/fake_homedir"
+	export CALIBRE_CONFIG_DIRECTORY="${HOME}/.config/calibre"
+	mkdir -p "${CALIBRE_CONFIG_DIRECTORY}" || die
+
+	addpredict /dev/dri #665310
+
+	# If this directory doesn't exist, zsh completion won't install
+	dodir /usr/share/zsh/site-functions
+
+	edo "${PYTHON}" setup.py install \
+		--staging-root="${ED}/usr" \
+		--prefix="${EPREFIX}/usr" \
+		--libdir="${EPREFIX}/usr/$(get_libdir)" \
+		--staging-libdir="${ED}/usr/$(get_libdir)" \
+		--system-plugins-location="${EPREFIX}/usr/share/calibre/system-plugins"
+
+	cp -r man-pages/ "${ED}"/usr/share/man || die
+
+	find "${ED}"/usr/share -type d -empty -delete || die
+
+	einfo "Converting python shebangs"
```

Comment:
I'd consider dropping the two `einfo` calls but they're OK if you want to keep them.